WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsOnboardingError.swift

```diff
@@ -26,7 +27,11 @@ struct InPersonPaymentsOnboardingError: View {
             if let buttonViewModel = buttonViewModel {
                 Button(buttonViewModel.text, action: buttonViewModel.action)
                     .buttonStyle(PrimaryButtonStyle())
-                    .padding(.bottom, 24.0)
+                    .padding(.bottom, secondaryButtonViewModel == nil ? 24.0 : 0)
```

Comment:
We could add the padding to the `secondaryButtonViewModel` (up). That way we don't have to check twice if `secondaryButtonViewModel` is nil

Reply:
That was my initial approach, but since `InPersonPaymentsOnboardingError` is reused in other views where this secondary button is nil, the padding attached to this view wouldn't render either. A good chance to refactor it is with the issue I share in your other comment, to extract some of the logic to the view model.